### DISCUSSION TITLE
Closing old PRs

### DISCUSSION THREAD

**Discussion by GintasS** [gitextensions/gitextensions#9441] [Ideas]
Hey, I have seen a lot of PR that are old and not really worked on. I think we should close them to clean the Pull requests section a little bit. The branches would stay.

I could do that, by going through them and making a mention for @RussKie or some other maintainer.

**Comment by gerhardol** [comment]
Setting them to a draft is sufficient, then they are not in the normal flow.
But before that, there should be an assessment if they are interesting at all or if there are some final touchups needed.

**Reply by RussKie** [reply]
Yes, I know... I guess we should close stale PRs.
For me those are like a wall of (my) shame. Some of those, whilst can't be merged, still can be built and iterated on (at least it was my dream). 
Realistically though I'm overcommitted at work, locked down at home schooling kids... yeah, I won't stand in a way if someone wants to start moping up.

**Reply by GintasS** [reply]
So ok, I can go through all of them and mark you or gerhardol.

**Reply by GintasS** [reply]
I have marked both you and gerhardol on old PRs, now we only need to close them and we would be good. Left out recent(ish) PRs, with recent activity, those who had activity less than 3~ months ago.

**Reply by GintasS** [reply]
Seems like that msftbot didn't close old PRs, but it should have.

**Reply by pmiossec** [reply]
Could we add a label like `cleaned_pr` to all this pr to find them again easily?
Because GitHub doesn't allow to differentiate closed PR and merged PR :( (they have all the status "closed")
So it is a pain to find them again if needed...

**Reply by gerhardol** [reply]
is:unmerged can be used
https://github.com/gitextensions/gitextensions/pulls?q=is%3Apr+is%3Aunmerged+is%3Aclosed

However, the reason for closing the PR is unknown so such a label could make sense.

**Comment by RussKie** [comment]
We could use "abandoned" maybe?
